Contract/contracts/ERC721.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol"; missing

Reply:
```suggestion
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol"; 
import "@openzeppelin/contracts/access/Ownable.sol";
```

Reply:
Hi Rhen, thanks for pointing this out, the classes ERC721 and ERC721URIStorage have the burn function, I don't need to import the Burnable, however, I noticed that I forgot to add a public method to burn by tokenId, I'm creating a new PR for that! thanks!

Reply:
Thanks for letting me know William!